boskos/janitor/BUILD.bazel

```diff
@@ -42,7 +42,7 @@ pkg_tar(
 
 container_image(
     name = "janitor_image",
-    base = "@gcloud-go//image",
+    base = "@cloud-sdk-slim//image",
```

Comment:
How do you know this will work?

Reply:
I've done some manual testing and it appears to work correctly.